bin/alluxio-fuse.sh

```diff
@@ -46,14 +45,8 @@ set_java_opt () {
     -Xmx1G
   "
 
-  ALLUXIO_FUSE_OPTS+="
-    -Dalluxio.logger.type=alluxio.fuse
-    -Dalluxio.master.port=${ALLUXIO_MASTER_PORT}
-    -Dalluxio.master.hostname=${ALLUXIO_MASTER_ADDRESS}
-    -Dalluxio.logs.dir=${ALLUXIO_LOGS_DIR}
-    -Dalluxio.logger.type="FUSE_LOGGER"
-    -Dlog4j.configuration=file:${ALLUXIO_CONF_DIR}/log4j.properties
-  "
+  ALLUXIO_FUSE_OPTS=${ALLUXIO_JAVA_OPTS}
```

Comment:
let's call this ALLUXIO_FUSE_JAVA_OPTS (for consistency with all other *_JAVA_OPTS)